packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -969,6 +1000,120 @@ export function mixinClientSelection<
 	};
 }
 
+export function mixinStashedClient<
+	TChannelFactory extends IChannelFactory,
+	TOperation extends BaseOperation,
+	TState extends DDSFuzzTestState<TChannelFactory>,
+>(
+	model: DDSFuzzModel<TChannelFactory, TOperation, TState>,
+	options: DDSFuzzSuiteOptions,
+): DDSFuzzModel<TChannelFactory, TOperation | StashClient, TState> {
+	if (options.clientJoinOptions?.stashableClientProbability === undefined) {
+		return model as DDSFuzzModel<TChannelFactory, TOperation | StashClient, TState>;
+	}
+
+	const generatorFactory: () => AsyncGenerator<TOperation | StashClient, TState> = () => {
+		const baseGenerator = model.generatorFactory();
+		return async (state): Promise<TOperation | StashClient | typeof done> => {
+			if (
+				!state.isDetached &&
+				"stashData" in state.client &&
+				state.client.stashData !== undefined &&
+				state.client.containerRuntime.pendingMessages.length > 0 &&
+				state.random.bool(0.5)
```

Comment:
Should this use stashableClientProbability, or is there a reason to use 0.5?

Reply:
stashableClientProbability is about creating the client, as we need to keep more data on those clients, so not all clients will be stashable.

once we get here, we are looking for a dirty client, which we determine by the `pendingMessages.length`, and which was created stashable, which already limits the number of potential states this could apply to pretty severely.  The 50% here just means we won't stash all stashable client the first time they are dirty, this allows more complicated stash patterns where some ops local ops are saved before stash. 

i did consider making this tunable, but i think it gets too complicated, so only left the knob to determine how often a client that can be stashed is create.